lib/jurnal_api/client/cores.rb

```diff
@@ -0,0 +1,12 @@
+module JurnalApi
+  class Client
+    # Defines methods related to Core
+    module Cores
+
+      def health_check(params = {})
+        response = get('core/ping/health_check', params)
```

Comment:
@kevinmekari could you please add spec for this endpoint?

Reply:
done mas @rslhdyt